### DISCUSSION TITLE
Backend team meeting on the 21st April

### DISCUSSION THREAD

**Discussion by Meminseeker** [bounswe/bounswe2024group11#199] [General]
As we talked earlier, we will meet this Sunday evening. What are your availabilities?

@yunuskaydin @ceydanursen @MucahitErdoganUnlu @ozankrkya @Meminseeker

**Comment by MucahitErdoganUnlu** [comment]
available after 7pm

**Comment by yunuskaydin** [comment]
Are we meeting today or on 22nd of April? I'm confused.

**Reply by Meminseeker** [reply]
It's my bad. I mistakenly wrote the wrong date. As discussed earlier, we are meeting today, the 21st of April.

I've corrected the title.

**Reply by yunuskaydin** [reply]
No problem. I will be outside but I'm able to take the meeting notes. I just can't talk too much. Time won't matter.

**Reply by MucahitErdoganUnlu** [reply]
we can do it at a late time. will it suit you? will you be at home?

**Reply by yunuskaydin** [reply]
I'll come home after 21:15 or so. If it suits our team that would be great.

**Comment by ceydanursen** [comment]
I am available after 19.30
